app/assets/javascripts/components/article_finder/article_finder_row.jsx

```diff
@@ -14,8 +14,9 @@ const ArticleFinderRow = createReactClass({
     };
   },
 
-  UNSAFE_componentWillReceiveProps(nextProps) {
-    if (this.state.isLoading && (this.props.assignment !== nextProps.assignment)) {
+  componentDidUpdate(prevProps, prevState) {
```

Comment:
This would benefit from a comment explaining how we're using `isLoading` for two different situations and so need to do this in a weird way that violates the usual best practices.

Reply:
Done!